### PR TITLE
Add action to test charts categories

### PR DESCRIPTION
**Pull request by miguelaeh**
**Description of the change**
This PR adds a GitHub action to test that the Helm Charts contains a valid category.
This PR completes https://github.com/bitnami/charts/pull/3075 but it is created separately to make the testing easier.

### CODE REVIEW

File: .github/actions/category-checker/category-checker.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+command=${1:?Missing command}
+
+ALLOWED_CATEGORIES=(Analytics ApplicationServer CMS CRM CertificateAuthority Database DeveloperTools Forum HumanResourceManagement Infrastructure LogManagement MachineLearning ProjectManagement Wiki WorkFlow E-Commerce eLearning)
+
+if [[ "$command" == "check-categories" ]]; then
+  modified_charts=($(ct --config /ct-config.yaml list-changed))
+  if [[ -z "$modified_charts" ]]; then
```
Comment (andresbono) [inline .github/actions/category-checker/category-checker.sh:9]:
```suggestion
  if [[ "${#modified_charts[@]}" -eq 0 ]]; then
```

File: .github/actions/category-checker/category-checker.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    echo "::set-output name=categories-are-correct::true"
+  else
+    success=false
+    for chart in "${modified_charts[@]}"; do
+      category=$(yq -r .annotations.category "bitnami/${chart}/Chart.yaml")
+      [[ ${ALLOWED_CATEGORIES[*]} =~ "$category" ]] && success=true
```
Comment (andresbono) [inline .github/actions/category-checker/category-checker.sh:15]: Is there only one possible category per chart?

Comment (andresbono) [inline .github/actions/category-checker/category-checker.sh:15]:
```suggestion
      category=$(yq -r .annotations.category "bitnami/${chart}/Chart.yaml")
      [[ "${ALLOWED_CATEGORIES[@]}" =~ "$category" ]] && success=true
```

File: .github/actions/category-checker/category-checker.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+command=${1:?Missing command}
+
+ALLOWED_CATEGORIES=(Analytics ApplicationServer CMS CRM CertificateAuthority Database DeveloperTools Forum HumanResourceManagement Infrastructure LogManagement MachineLearning ProjectManagement Wiki WorkFlow E-Commerce eLearning)
```
Comment (andresbono) [inline .github/actions/category-checker/category-checker.sh:5]:
```suggestion
ALLOWED_CATEGORIES=(
  "Analytics"
  "ApplicationServer"
  "CertificateAuthority"
  "CMS"
  "CRM"
  "Database"
  "DeveloperTools"
  "E-Commerce"
  "eLearning"
  "Forum"
  "HumanResourceManagement"
  "Infrastructure"
  "LogManagement"
  "MachineLearning"
  "ProjectManagement"
  "Wiki"
  "WorkFlow"
)
```

File: .github/actions/category-checker/category-checker.sh (hunk 2 above)
Comment (miguelaeh) [reply .github/actions/category-checker/category-checker.sh:15]: yes

**Pull request merged**